src/backend/services/mining-api/src/MiningPool.py

```diff
@@ -0,0 +1,53 @@
+import threading
+import time
+import queue
+
+class MiningPool:
+    DEQUEUE_TIME = 5
+
+    def __init__ (self, receiver, ready_to_mine):
+        self._pool = queue.Queue()
+        self._mining_thread = threading.Thread(target=self.sendToMine, kwargs={'self': self, 'receiver': receiver})
+        self._mining_thread.daemon = True
+
+        self._send = threading.Condition()
+        self._ready_to_mine = ready_to_mine     
+
+    def addToPool(self, data):
+        self._pool.put(data)
+
+    def start_thread(self):
```

Comment:
I may be missing it but I don't see where this gets called. If that's the case, does it need to be called or can we remove it? Should be called in init I'd think